pkg/volume/emptydir/quota.go

```diff
@@ -84,6 +94,11 @@ func (cr *realQuotaCommandRunner) RunApplyQuotaCommand(fsDevice string, quota re
 	return "", stderr.String(), err
 }
 
+func (cr *realQuotaCommandRunner) RunMountOptionsCommand() (string, string, error) {
```

Comment:
It looks like the second string field in the return value is never used.  Do we need it?